src/macaron/slsa_analyzer/provenance/witness/__init__.py

```diff
@@ -133,28 +134,25 @@ def extract_witness_provenance_subjects(witness_payload: InTotoPayload) -> set[W
 
     Returns
     -------
-    dict[str, str]
+    list[InTotoV01Subject]
         A dictionary in which each key is a subject name and each value is the corresponding SHA256 digest.
```

Comment:
Need to update comment.

Reply:
Also need to remove non-existent `extensions` parameter from comment above.

Reply:
Thanks. Fixed in 9e6f98f27a3e7255e6b5333907ad3b626e2220a8.